src/makeCalculator.js

```diff
@@ -32,12 +32,42 @@
  *  .operate(calculator.multiply, 7)
  *
  * calculator.result === -28
- *
- *
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      return calculator.result + number;
```

Comment:
here and in other methods better to use this keyword
```suggestion
      return this.result + number;
```